pkg/io/jsonio/unmarshal.go

```diff
@@ -28,6 +29,12 @@ func (*Reader) ReadSpec(filename string, configDefs *ir.ConfigDefs) (*ir.Spec, e
 		return nil, err
 	}
 
+	cidrSegments := translateSegments(jsonspec.Segments, TypeCidr)
+	finalCidrSegments, err := computeContainedSubentsInCidrSegment(configDefs, cidrSegments)
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
Move below the initialization of `configDefs`, and initialize appropriately. This should help run tests with custom address prefixes.